### PR TITLE
 Update info about categories returned by Perceval

### PR DESCRIPTION
**Pull request by sarvesh211999**
This is in reference to the issue [here](https://github.com/chaoss/grimoirelab-perceval/issues/494)

Discussion about this is done in this [PR ](https://github.com/chaoss/grimoirelab-perceval/pull/495)

The information about cartegories returned by Perceval is not mentioned.
Note added: Every Pull Request is Issue , but not every issue is a pull request

### CODE REVIEW

File: perceval/github.md [hunk 1]
```diff
@@ -16,6 +16,17 @@ In that help banner, we see two different ways of using it:
...
 * with a user token
 
+## Retrieving from GitHub with category param
+
+```bash 
+(perceval) $ perceval github --category issue grimoirelab perceval
+...
+```
+
+You can pass category as parameter which take issue,pull_request,commit and repository when using Github Perceval backend.
```
Comment (inishchith) [inline perceval/github.md]: @sarvesh211999 Just a minor comment. `GitHub` backend only accepts either of these (3) `[ issue, pull_request, repository ]` as parameter. 

[REF](https://github.com/chaoss/grimoirelab-perceval/blob/b3031c06bf76f5380db661a7c6c7be590499199c/perceval/backends/core/github.py#L90)

Comment (sarvesh211999) [reply perceval/github.md]: I will change the wording.

**Comment by valeriocos** [comment]
thank you @sarvesh211999 and @inishchith for the PR and review.

**Comment by sarvesh211999** [comment]
@valeriocos is there any changes required?

**Comment by valeriocos** [comment]
@jgbarah the PR is ready to merge. I can do the merge if you want.

**Pull request merged**